pkg/build/builder/cmd/builder.go

```diff
@@ -222,14 +226,24 @@ func (c *builderConfig) execute(b builder) error {
 	}
 	glog.V(4).Infof("Running build with cgroup limits: %#v", *cgLimits)
 
-	if err := b.Build(c.dockerClient, c.dockerEndpoint, c.buildsClient, c.build, cgLimits); err != nil {
-		return fmt.Errorf("build error: %v", err)
+	if c.build.Spec.Strategy.SourceStrategy != nil {
+		glog.Infof("Build is using %s as the s2i builder image", c.build.Spec.Strategy.SourceStrategy.From.Name)
+	}
+	if c.build.Spec.Strategy.DockerStrategy != nil {
+		if c.build.Spec.Strategy.DockerStrategy.From != nil {
+			glog.Infof("Build is using %s as the base FROM image", c.build.Spec.Strategy.DockerStrategy.From.Name)
+		}
 	}
 
+	err = b.Build(c.dockerClient, c.dockerEndpoint, c.buildsClient, c.build, cgLimits)
+
 	if c.build.Spec.Output.To == nil || len(c.build.Spec.Output.To.Name) == 0 {
 		fmt.Fprintf(c.out, "Build complete, no image push requested\n")
 	}
 
+	if err != nil {
+		return fmt.Errorf("build error: %v", err)
+	}
```

Comment:
Do we want the "Build complete, no image push..." message to appear even if there is a build error? I'm inclined to say no.

Reply:
yeah this got reordered, i'll fix it, thanks.